src/main/java/org/fundacionjala/core/ui/browsers/GoogleChrome32.java

```diff
@@ -15,6 +15,7 @@
      * Empty constructor and path fof driver.
      */
     public GoogleChrome32() {
+        //TODO multi platform support
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)